README.md

```diff
@@ -21,7 +21,7 @@ _This list is for projects, tools, or pretty much any things related to IPFS tha
 
 ## Contribute to this list!
 
-Everyone is welcome to submit their new awesome-ipfs item. Check the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md) to learn how to do so.
+Everyone is welcome to submit their new awesome-ipfs item. In order to add an element to this list, you need to modify the files in `/data` and then run a simple `make build` before publishing your pull request as the Readme and the website are automatically generated. Check the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md) to learn how to do so.
```

Comment:
I think this is already described in the `CONTRIBUTING.md`, isn't it? 

> Make sure to run `npm run build:readme` or `yarn build:readme` before submitting the Pull Request

Reply:
Looking at the issue, seems things are not clear or direct enough. I think we just have to mention it's auto generated from a script, and to properly make changes, people have to read the contributing file. Right now, we're just duplicating the instructions and making this section bigger, which means less people will actually read it.

Reply:
yes, that is why I made that change.
It is not duplicating but making it clear IMHO. No? 
@Mr0grog what do you think?